exporter/prometheusexporter/config.go

```diff
@@ -36,12 +38,48 @@ type Config struct {
 	EnableOpenMetrics bool `mapstructure:"enable_open_metrics"`
 
 	// AddMetricSuffixes controls whether suffixes are added to metric names. Defaults to true.
+	// Deprecated: Use TranslationStrategy instead when exporter.prometheusexporter.UseTranslationStrategy feature gate is enabled.
 	AddMetricSuffixes bool `mapstructure:"add_metric_suffixes"`
+
+	// TranslationStrategy controls how OTLP metric and attribute names are translated into Prometheus metric and label names.
+	// Only used when exporter.prometheusexporter.UseTranslationStrategy feature gate is enabled.
+	TranslationStrategy translationStrategy `mapstructure:"translation_strategy"`
 }
 
 var _ component.Config = (*Config)(nil)
 
 // Validate checks if the exporter configuration is valid
-func (*Config) Validate() error {
+func (cfg *Config) Validate() error {
+	if translationStrategyFeatureGate.IsEnabled() {
+		switch cfg.TranslationStrategy {
+		case underscoreEscapingWithSuffixes, underscoreEscapingWithoutSuffixes, noUTF8EscapingWithSuffixes, noTranslation:
+		default:
+			return fmt.Errorf("invalid translation_strategy: %s", cfg.TranslationStrategy)
+		}
+	}
 	return nil
 }
+
+type translationStrategy string
+
+const (
+	// underscoreEscapingWithSuffixes fully escapes metric names for classic Prometheus metric name compatibility,
+	// and includes appending type and unit suffixes
+	underscoreEscapingWithSuffixes translationStrategy = "UnderscoreEscapingWithSuffixes"
+
+	// underscoreEscapingWithoutSuffixes escapes special characters to '_', but suffixes won't be attached
+	underscoreEscapingWithoutSuffixes translationStrategy = "UnderscoreEscapingWithoutSuffixes"
+
+	// noUTF8EscapingWithSuffixes disables changing special characters to '_'. Special suffixes like units and '_total' for counters will be attached
+	noUTF8EscapingWithSuffixes translationStrategy = "NoUTF8EscapingWithSuffixes"
+
+	// noTranslation bypasses all metric and label name translation, passing them through unaltered
+	noTranslation translationStrategy = "NoTranslation"
+)
+
+var translationStrategyFeatureGate = featuregate.GlobalRegistry().MustRegister(
```

Comment:
IMO the feature gate should only control the removal of existing configuration options, and translation_strategy should always take precedence over existing conflicting options.  That way someone can just migrate and doesn't need to mess with feature gates.